pages/tasks/team_details.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+from pages.tasks.regions.task import Task
+
+
+class TeamDetailsPage(Base):
+
+    _name_locator = (By.CSS_SELECTOR, '.content-container > h2')
+    _task_list_header_locator = (By.CSS_SELECTOR, 'div.task-list-container > h3')
+    _available_tasks_list_locator = (By.CSS_SELECTOR, '.task-list > li')
+
+    @property
+    def _page_title(self):
+        return '%s | Mozilla One and Done' % self.name
+
+    @property
+    def name(self):
+        # need to remove ' Team' from the element
+        return self.selenium.find_element(*self._name_locator).text[:-5]
```

Comment:
Yeah, I wasn't enthused by this either... Do you mean the title element of the page header or the h2 in this class? I'm using the name link from the task page as the source of truth, and the name appears three times on the team detail page but not in an easy to access format (title and two headers).